### PR TITLE
Spin (mobile): Applied UX guide for the Spin widget

### PR DESCRIPTION
**Pull request by HunseopJeong**
[Issue] N/A
[Problem] Default style for the spin was suitable for the wearable style
[Solution] Applied the UX guide for the Spin

Signed-off-by: Hunseop Jeong <hs85.jeong@samsung.com>

### CODE REVIEW

**Comment by HunseopJeong** [comment]
image
There is no detaul UX guide for the Spin.
I modified only the parts that needed to be changed as much as possible and kept many parts.

**Review by singa2000 — APPROVED** [review]
It looks good.

File: src/js/core/widget/core/Spin.js [hunk 1]
```diff
@@ -189,6 +189,7 @@
 					item.style.transform = "translateY(-1000px)"; // move item from active area
 				}
 				item.style.opacity = change.opacity;
+				item.style.height = itemHeight + "px";
```
Comment (DohyungLim) [inline src/js/core/widget/core/Spin.js:192]: This code is not available in wearable. What is the effect on the item?

**Review by DohyungLim — COMMENTED** [review]
1 comment.

File: src/js/core/widget/core/Spin.js (hunk 1 above)
Comment (HunseopJeong) [reply src/js/core/widget/core/Spin.js:192]: I checked in wearable profile.
It does not set the their height if useer change the itemHeight value.
So I added it.

**Review by DohyungLim — APPROVED** [review]
LGTM.

**Pull request merged**